spec/system/footer_spec.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+RSpec.describe "Footer", system: true do
+  let!(:theme) { upload_theme_component }
+
+  it "should display the footer" do
+    theme.update_setting(:Link_sections, "Section 1, Section 1 title|Section 2, Section 2 title")
```

Comment:
Oh maybe the capitalization is a theme setting thing generally? Seems strange to me, but not related to this PR